.github/workflows/ci.yml

```diff
@@ -0,0 +1,72 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: [8.1]
+        db_image_name: [mysql, mariadb, percona, postgres]
+        include:
+          - db_image_name: mysql
+            db_image_version: 8
+            db: mysql
+            vars: '-e MYSQL_PASSWORD=travis -e MYSQL_DATABASE=doctrine_extensions_tests -e MYSQL_USER=travis -e MYSQL_ROOT_PASSWORD=travis'
```

Comment:
Please check the DB credentials, it seems that the ones used are actual for travis CI.

Reply:
I don't understand what you expect from me.

Do you want me to switch to another User / Password?
Is Travis CI still used anywhere?

Reply:
Travis CI is no longer used.

 > Do you want me to switch to another User / Password?

Probably yes, the mention of the Travis looks a bit strange here.